hello/views.py

```diff
@@ -1,3 +1,28 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from .models import News, Author
 
-# Create your views here.
+# View for the home page where the latest 10 news articles are displayed
+def home(request):
+    latest_news = News.objects.order_by('-pub_date')[:5]  # Fetch the latest 10 news articles
```

Comment:
use limit instead of [:5]. 
For now, you fetch all records from the DB instead of fetching only those you need